pkg/util/http.go

```diff
@@ -22,6 +22,30 @@ import (
 
 const messageSizeLargerErrFmt = "received message larger than max (%d vs %d)"
 
+const (
```

Comment:
We have that already defined here

https://github.com/grafana/tempo/blob/303c7ef9e086955071312522de5c1bc26565e733/pkg/api/http.go#L66